opennms-webapp-rest/src/main/java/org/opennms/web/rest/v1/FilesystemRestService.java

```diff
@@ -98,8 +98,9 @@ public List<String> getFiles(@QueryParam("changedFilesOnly") boolean changedFile
         }
 
         try {
-            return Files.find(etcFolder, 4, (path, basicFileAttributes) -> isSupportedExtension(path))
```

Comment:
Silly question, but can't you just change the `Files.find` to have `FOLLOW_SYMLINKS` on the end of it using the 4-arg version?  This means even if someone symlinks around other things deeper in the tree, it would still work without having to do the extra map.

```java
return Files.find(etcFolder, 4, (path, basicFileAttributes) -> isSupportedExtension(path), FileVisitOption.FOLLOW_LINKS)
```

Reply:
tested all working fine.